### PR TITLE
Release Notes

### PR DESCRIPTION
**Pull request by biodranik**
Please insert any translations as comments, not code suggestions. It will be easier to replace.

### CODE REVIEW

File: android/app/src/fdroid/play/listings/ar/release-notes.txt [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
-• Parking entrances are searchable
-• Test TTS voice in the settings
-• Fixed arrow position button on Android 5&6
+• New OpenStreetMap data as of November 13
+• It is possible to zoom in deeper now and see more objects in dense areas
+• Outdoors map style preview (activate with ?olight or ?odark in search)
+• Fixed import of bookmarks in KMB format
+• Corrected the misleading location button's icon state
+• "Inertial" kinetic scroll improvements
+• Improved routing, search, styles, icons, and translations
```
Comment (rtsisyk) [inline android/app/src/fdroid/play/listings/ar/release-notes.txt:7]: > "Corrected the misleading location button's icon state"
> "Improved routing, search, styles, icons, and translations" 

are too generic to understand what has changed. It would like to have these three items in the list:

- Fixed the missing 'green dot' location indicator at the right top of the Android screen
- Disabled the screen wake lock by default to fix power-saving when using the app on a locked screen
- Corrected the misleading location button's icon state when the map is moved or zoomed during a location search

Please note that the proposed text above is more specific. The user can actually validate the change and see the difference. 

> It is possible to zoom in deeper now and see more objects in dense areas

This one is also too generic. Even I don't understand what has changed. Any examples?

Comment (pastk) [reply android/app/src/fdroid/play/listings/ar/release-notes.txt:7]: Good points, but your suggested text won't fit into Android's characters limit...

Also I personally have no idea what is a "green dot location indicator". Never saw one. I have a small pin in the status bar when there is a fix. Maybe depends on android's version and/or brand.

Re "possible to zoom deeper": its about z20 addition. Now max zoom is twice more / deeper than it was before.
How would you formulate it?

Comment (pastk) [reply android/app/src/fdroid/play/listings/ar/release-notes.txt:7]: > Improved routing, search, styles, icons, and translations

"Many other style, routing, etc. improvements, see full list on omaps.org/news"

Comment (rtsisyk) [reply android/app/src/fdroid/play/listings/ar/release-notes.txt:7]: > Also I personally have no idea what is a "green dot location indicator". 

Ok, let me rephrase.

Comment (d4f5409d) [reply android/app/src/fdroid/play/listings/ar/release-notes.txt:7]: Hmu if you've done it and I'll start translating to 🇭🇺

Comment (rtsisyk) [reply android/app/src/fdroid/play/listings/ar/release-notes.txt:7]: give me 10 mintues.

Comment (rtsisyk) [reply android/app/src/fdroid/play/listings/ar/release-notes.txt:7]: • New OpenStreetMap data as of November 13
• The maximum zoom level is twice as deep as it was before.
• Fixed import of bookmarks from MAPS.ME app
• The preview of outdoor map style - activate with ?olight or ?odark in search
• Disabled the screen wake lock by default to improve power-saving
• Improved the location search indication in both the system and the app.
• "Inertial" kinetic scroll improvements
• Many other style, routing, etc. improvements, see full list on omaps.org/news


@pastk @biodranik PTAL

Comment (pastk) [reply android/app/src/fdroid/play/listings/ar/release-notes.txt:7]: As you've merged #6523 then it should be added to the release note too (perhaps instead of something else less important).

File: android/app/src/fdroid/play/listings/de-DE/release-notes.txt [hunk 2]
```diff
@@ -1,13 +1,9 @@
...
-• Neue Distanzmessung von geraden Linien
-• Hintergrundstimme und Wegbeschreibungen
-• Fehler "Warten..." Dialog bei Android 13 mit SD-Karte behoben
-• Musik wird bei Bildschirmdrehung nicht mehr pausiert
-• Icons aktualisiert
-• Verbesserte Relevanz der Suchergebnisse
-• Parkhauseingänge sind suchbar
-• Sprachtest im Einstellungsmenü
-• Fehler bei backurl API behoben
-• Verbesserte arabische, polnische und spanische Übersetzung
```
Comment (JohannesDeml) [inline android/app/src/fdroid/play/listings/de-DE/release-notes.txt:11]: • Neue OSM-Daten mit Stand 13. November
* Es ist jetzt möglich weiter in die Karte zu zoomen um mehr Objekte in dichten Gebieten zu sehen
* Outdoor Karten Anzeige Vorschau (aktivierbar mit den Begriffen ?olight oder ?odrak in der Suchleiste)
* Probleme für den Import von Lesezeichen im KBM Vormat behoben
* Verbesserung der Anzeige des Standort-Knopfs
* Verbesserungen beim Scrollen von Karten
* Verbesserte Wegfindung, Suche, Anzeige, Icons und Übersetzungen

File: android/app/src/fdroid/play/listings/ar/release-notes.txt (hunk 1 above)
Comment (rtsisyk) [reply android/app/src/fdroid/play/listings/ar/release-notes.txt:7]: • 2023-11-13 OpenStreetMap data
• The maximum zoom level is twice as deep as it was before
• Outdoors map style preview (activate with ?olight or ?odark in the search)
• Fixed import of bookmarks from MAPS.ME app
• Added experimental "Details on KAYAK" affiliate link to hotels
• Disabled the screen wake lock by default to improve power-saving
• Improved the location search indication in both the system and the app
• Many other style, routing, etc. improvements, see full list on omaps.org/news

Comment (pastk) [reply android/app/src/fdroid/play/listings/ar/release-notes.txt:7]: > Outdoors map style preview (activate with ?olight or ?odark in the search)

I'm afraid people will activate it and won't know how to deactivate :)

• New OpenStreetMap data as of November 13
• The maximum zoom level is twice as deep as it was before
• Fixed import of bookmarks from MAPS.ME app
• Added experimental "Details on KAYAK" affiliate link to hotels
• Disabled the screen wake lock by default to improve power-saving
• Corrected the location search indication when the map is moved or zoomed
• Outdoors map style preview - read on how to activate it and about many other improvements on omaps.org/news

480 chars

Comment (rtsisyk) [reply android/app/src/fdroid/play/listings/ar/release-notes.txt]: Updated to @pastk's text.

**Comment by rtsisyk** [comment]
Sorry for the short notice, but I plan to push this release in the next hour. Otherwise it probably will miss the last window on Friday and stuck on review until Monday or Tuesday.

**Review by rtsisyk — APPROVED** [review]

File: iphone/metadata/th/release_notes.txt [hunk 3]
```diff
@@ -1,30 +1,54 @@
-• New OSM maps data as of September 20
-• Added new route type "Ruler" (aka Helicopter Routing) to quickly measure straight distances
-• Set decimal and grouping separators depending on the system locale
-• Removed annoying zoom in route planning mode
-• Fixed Open/Closed shown sometimes in gray color
-• Caves can be added in the Editor
-• Improved Arabic, Polish, Spanish translations
+• New OpenStreetMap data as of November 13, 2023
+• It is possible to zoom in deeper now and see more POIs in dense areas
+• Outdoors map style preview (activate with ?olight or ?odark in search)
```
Comment (pastk) [inline iphone/metadata/th/release_notes.txt:3]: * Outdoors map style preview (please read on to know how to activate it)

**Review by pastk — APPROVED** [review]

File: iphone/metadata/uk/release_notes.txt [hunk 4]
```diff
@@ -1,30 +1,54 @@
...
-• Зменшено видимість пожежних гідрантів
-• Номери будинків відображаються над іконками та написами основних POI, а також не відображаються номери будинків для будівель, які надто малі для поточного масштабу
-• Не відображати піктограму зірки для квадратів
-• Уточнити шрифт номерів будинків та пріоритет малювання
-• Додані лінії пішохідних, мостових та велосипедних доріжок на детальному масштабі (z18+) у режимі навігації
-• Приховані пішохідні тунелі в режимі навігації
+Search
+• Process platz-pl like strasse-str
+• Improved address matching
+• Improved search of brands
```
Comment (strump) [inline iphone/metadata/uk/release_notes.txt:23]:
```suggestion
Пошук
• Адреси platz-pl обробляються як strasse-str
• Покращення роспазнавання адрес
• Покращення пошуку за брендом
```

**Review by strump — COMMENTED** [review]
Added Ukrainian translation (not complete)

File: iphone/metadata/uk/release_notes.txt [hunk 5]
```diff
@@ -1,30 +1,54 @@
...
+• Increased scrubs, alpine huts visibility
+• Updated sanitary dump station icons and adjusted its priority
+• Reduced visibility of cave entrances and improved the icon
+• Increased visibility of standalone house numbers
+• Reduced Tokyo subway visibility -- kudos to charlie2clarke
+• Removed minor road markers/shields cluttering the map in Estonia
+• Fix colors of Estonia and Germany road markers / shields
+• Increased visibilities of road markers / shields on tertiary and residential road
+• Fixed "gaps" in some densely mapped lines
+• Fixed "gaps" in the outlines of bridges and other compound lines
```
Comment (strump) [inline iphone/metadata/uk/release_notes.txt:41]:
```suggestion
Стиль мапи
• На мапі відображаються клумби, арендні магазини, оренда велосипедів, ремонт мотоциклів, контактні зоопарки, території вʼязниць, компресори для накачування коліс
• В режимі автомобільної навігації відображаються заповідники та військові зони
• Нові схеми метро Docklands Light Railway, та Porto metro
• Для лижних трас відображається назва
• Нові поліпшені іконки для фітнес-залів, центрів йоги, театрів, витворів мистецтва, скульптур, парків розваг, барбекю, маяків, вітряків, гелікоптерних майданчиків, мисливських стендів, Залів Царства Свідків Єгови, платних та бехкоштовних паркінгів, міні-магазінив, гастрономів, фермерських магазинів, продуктових магазинів, магазинів здорового харчування, тютюнових магазинів
• Покращено зовнішній вигляд господарських ділянок та гаражних кооперативів
• Краще видно лікарні та альпійські хатинки
• Оновлено іконку станції санітарного звалища та змінено пріоритет відображення
• Входи до печер відображаються рідше та отримали нову іконку
• Номери будинків навпаки відображаються частіше
• Іконки токійського метро більше не засмічують мапу. Подяка за це charlie2clarke
• Прибрано щитки другорядних доріг в Естонії
• Виправлено колір дорожніх щитків в Німечині та Естонії
• Покращено видимість дорожніх щитків для третьорядних та міських доріг
• Виправлені прогалини в деяких місцях із великою кількістю ліній
• Виправлені прогалини в полігонах мостів та інших зʼєднувальних ліній
```

File: iphone/metadata/uk/release_notes.txt [hunk 6]
```diff
@@ -1,30 +1,54 @@
...
+
+Outdoors map style preview:
+• Tailored for out-of-the-city outdoor activities like hiking, backcountry cycling/MTB, 4x4, etc.
+• Emphasizes paths/tracks, cliffs, railways, powerlines, rivers and streams, water sources, peaks, mountain saddles and passes, waterfalls, lighthouses, caves, viewpoints, campsites, huts, shelters, picnic sites, gates, stiles, fords, etc.
+• Enable by typing `?olight` in the search field and return to the map; type `?light` to switch back to the main style
+• Use `?odark` and `?dark` for a dark style
+• Android and iOS developers wanted to help with adding a nice UI for style switching!
```
Comment (strump) [inline iphone/metadata/uk/release_notes.txt:48]:
```suggestion
Новий стиль для орієнтування на природі:
• Пристосований для тих, кто проводить час за містом, ходить в походи, їздить по пересіченій місцевості на велосипедах чи позашляховиках
• Акцентує увагу на пішохідних стежках, кручах, залізничних дорогах, лініях електропередач, річках та струмочках, джерелах води, вершинах, гірських перевалах та стежках, водопадах, маяках, печерах, оглядових майданчиках, кемпінгах, хатинках, укриттях, місцях для пікніку, воротах, парканах, тощо
• Щоб увімкнути новий стиль введіть `?olight` в поле пошуку та поверніться до мапи. Введить `?light` щоб повернути стандартний стиль
• Використовуйте `?odark` та `?dark` для темного режиму
• Закликаємо Android та iOS розробників реализувати більш зручний UI для перемикання природного стилю!
```

File: iphone/metadata/uk/release_notes.txt [hunk 7]
```diff
@@ -1,30 +1,54 @@
...
+
+Translations
+• Improved "avoid …" routing options wording
+• Improved Arabic, Basque, Chinese, Czech, Danish, Finnish, Greek, Hungarian, Indonesian, Italian, Korean, Marathi, Polish, Romanian, Slovak, Spanish, Sweden, Thai, Turkish, Ukrainian, and Vietnamese translations
+
+…and more smaller fixes and improvements, check https://omaps.org/news for details!
```
Comment (strump) [inline iphone/metadata/uk/release_notes.txt:54]:
```suggestion
Переклади
• Покращено текст при виборі правил обʼїзду
• Покращено переклади арабською, баскською, китайською, чеською, датською, фінською, грецькою, угорською, індонезійською, італійською, корейською, польською, румунською, словацькою, іспанською, шведською, тайською, турецькою, українською та вʼєтнамською мовами

… та ще багато невеликих поліпшень. Слідкуйте за новинами https://omaps.org/news
```

**Review by strump — COMMENTED** [review]
Finished iOS release notes translation
